dotcom-rendering/src/web/components/HeaderTopBar.importable.tsx

```diff
@@ -64,10 +67,13 @@ export const HeaderTopBar = ({
 				/>
 				<SearchJobs />
 
-				<Search
-					href="https://www.google.co.uk/advanced_search?q=site:www.theguardian.com"
-					dataLinkName="nav3 : search"
-				/>
+				{!headerTopBarSearchCapiSwitch && (
+					<Search
+						href="https://www.google.co.uk/advanced_search?q=site:www.theguardian.com"
+						dataLinkName="nav3 : search"
+					/>
+				)}
+				{headerTopBarSearchCapiSwitch && <HeaderTopBarSearchCapi />}
```

Comment:
I think a ternary would be clearer, here?

```suggestion
				{headerTopBarSearchCapiSwitch ? (
					<HeaderTopBarSearchCapi />
					) : (
					<Search
						href="https://www.google.co.uk/advanced_search?q=site:www.theguardian.com"
						dataLinkName="nav3 : search"
					/>
				)}
```